docs/deployment/feature_gates.md

```diff
@@ -73,3 +74,4 @@ A *General Availability* (GA) feature is also referred to as a *stable* feature.
 * `MountHostCADirectories` enables mounting common CA certificate directories in the Shoot API server pod that might be required for webhooks or OIDC.
 * `SeedChange` enables updating the `spec.seedName` field during shoot validation from a non-empty value in order to trigger shoot control plane migration.
 * `SeedKubeScheduler` adds custom `kube-scheduler` in `gardener-kube-scheduler` namespace. It schedules [pods with scheduler name](../concepts/seed-admission-controller.md#mutating-webhooks) `gardener-kube-scheduler` on Nodes with higher resource utilization. It requires Seed cluster with kubernetes version `1.18` or higher.
+* `BootstrapTokenForVMs` enables a security enhancement for the bootstrap flow of the kubelet. The bootstrap-token is now uniquely generated for each VM and invalidated after successful creation. This requires support in the worker controller (i.e. machine-controller-manager >`v0.26.0`) and in the os-extension (i.e. gardener-extension-os-gardenlinux >`v.0.9.0`) 
```

Comment:
What about rather calling this feature gate `BootstrapTokenProvidedByWorker` or something similar to make it more clear?